docs/source/usage/wm-widgets.rst

```diff
@@ -50,24 +50,24 @@ Add the following to :file:`~/.config/qtile/config.py`:
        ),
    ]
 
-.. _bar-usage:
+.. _lemonbar-usage:
 
-bar-aint-recursive
-==================
+lemonbar (formerly bar-aint-recursive)
+======================================
 
-To run the bar simply pipe the output of the binding script into ``bar`` and 
+To run the bar simply pipe the output of the binding script into ``lemonbar`` and 
 specify appropriate options, for example like this::
 
-    python /path/to/powerline/bindings/bar/powerline-bar.py | bar
+    python /path/to/powerline/bindings/lemonbar/powerline-lemonbar.py | lemonbar
 
 to run with i3, simply ``exec`` this in i3 config file::
 
-    exec python /path/to/powerline/bindings/bar/powerline-bar.py --i3 | bar
+    exec python /path/to/powerline/bindings/lemonbar/powerline-lemonbar.py --i3 | lemonbar
```

Comment:
Documentation is now invalid because the way lemonbar is invoked changed.

By the way, you say “lemonbar” here, but in the code it is “bar-aint-recursive”. Why?